java/spacewalk-java.changes

```diff
@@ -1,3 +1,7 @@
+- trigger reboot needed message also when installhint is available
+  on package level
+- add CLM filter for package provides and use it in live patching
```

Comment:
is `CLM` clear enough for customers? shouldn't it be expanded in the changelog to be more easily understood?